README.md

```diff
@@ -37,8 +37,9 @@ running:
 
     ./console-service -gen-token
 
-The OBMd web protocol can be "http" or "https".  If https is chosen,
-the "TLSCert" and "TLSKey" fields must be filled in.
+The OBMd web protocol can be insecure (http) or not (https).
+If `Insecure` is `false`, the `TLSCert` and `TLSKey` fields must be filled in.
+If `Insecure` is `true`, be sure to leave the TLS fields blank.
```

Comment:
The wording of this feels a bit odd to me, since the point of the  change was to make http a second-class option; I'm going to just send you a pr with a docs patch.